### PR TITLE
docs: 80 column lines (where possible), drop trailing whitespace

### PR DESCRIPTION
**Pull request by domo141**
with one typofix

### CODE REVIEW

File: docs/labwc-config.5.scd [hunk 1]
```diff
@@ -300,14 +300,14 @@ extending outward from the snapped edge.
...
+	Show a preview when snapping to a window to an edge. Default is yes.
 
 *<snapping><preview><delay><inner>*++
 *<snapping><preview><delay><outer>*
-	Sets the delay to show a preview when snapping a window to each type of edges.
-	Defaults are 500 ms.
-	*inner* edges are edges with an adjacent output and *outer* edges are edges
-	without an adjacent output.
+	Sets the delay to show a preview when snapping a window to each type of
+	edges. Defaults are 500 ms.
```
Comment (Consolatis) [inline docs/labwc-config.5.scd:308]: I know that it is not an issue introduced by this PR but would you up for also changing this to `[..] to each type of edge.` (singular).

**Comment by Consolatis** [comment]
Thanks, looks good to me. Just one small nitpick above.

**Comment by tokyo4j** [comment]
Thank you! I should've checked for typo and grammar more carefully.

**Comment by Consolatis** [comment]
> Thank you! I should've checked for typo and grammar more carefully.

No worries, it also passed the review.
Things like this happens and the beauty of open source allows others to detect and fix it :)

**Comment by domo141** [comment]
force-pushed s/edges./edge./ change I hope i get it right, not skilled enough to do "diffdiff" ;/

**Comment by Consolatis** [comment]
> force-pushed s/edges./edge./ change I hope i got it right, not skilled enough to do "diffdiff" ;/ here in gh

Yep, looks like you did everything right.
You can also just add another commit next time, when merging a PR we can automatically squash commits if necessary.

Thanks :)

**Pull request merged**